prow/plugins/config.go

```diff
@@ -621,6 +633,22 @@ func (c *Configuration) TriggerFor(org, repo string) Trigger {
 	return Trigger{}
 }
 
+// DcoFor finds the Dco for a repo, if one exists
+// a Dco can be listed for the repo itself or for the
+// owning organization
+func (c *Configuration) DcoFor(org, repo string) *Dco {
+	if org == "" && repo == "" {
+		return c.Dco["*"]
+	}
+	if c.Dco[fmt.Sprintf("%s/%s", org, repo)] != nil {
+		return c.Dco[fmt.Sprintf("%s/%s", org, repo)]
+	}
+	if c.Dco[org] != nil {
+		return c.Dco[org]
+	}
+	return c.Dco["*"]
```

Comment:
Can youn add a check if `c.DCO["*"]` is nil and if yes, return a `&DCO{}`? Consumers of this func will not expect to get a nilpointer back

Reply:
Added